app_controller/pixi_frontend/src/ts/init_pixi.ts

```diff
@@ -1,17 +1,67 @@
 import * as PIXI from "pixi.js"
+window.PIXI = PIXI;
```

Comment:
Why do you have to do this?

Reply:
You don't. I was debugging something and one of the advice was 'hey put this line'. It didn't work and I forgot to remove it.